config/sync/novel.settings.yml

```diff
@@ -0,0 +1,11 @@
+features:
+  node_user_picture: false
+  comment_user_picture: true
+  comment_user_verification: true
+  favicon: 1
+logo:
+  use_default: 1
+favicon:
+  use_default: 0
+  path: 'public://favicon.ico'
+  mimetype: image/vnd.microsoft.icon
```

Comment:
Please reconsider these values.

As far as I can tell `public://favicon.ico` is the path to a file that you have uploaded yourself. This will not work across deployments. As far as I can tell you have not added a new icon as a part of this PR.

I think it should be sufficient to disable the "Use the favicon supplied by the theme" setting in the theme configuration and then export the configuration.

If you feel like providing a default favicon with the theme you should be able to provide one by placing a `favicon.ico` file in the root of the Novel theme folder.